src/main/java/uk/gov/hmcts/divorce/document/content/ServiceOrderTemplateContent.java

```diff
@@ -69,9 +76,16 @@ public Map<String, Object> apply(final CaseData caseData,
 
         templateContent.put(CASE_REFERENCE, caseData.formatCaseRef(ccdCaseReference));
 
-        final LanguagePreference applicantLanguagePreference = caseData.getApplicant1().getLanguagePreference();
+        final LanguagePreference applicantLanguagePreference = caseData.isJudicialSeparationCase()
```

Comment:
This one can be removed and we can call the language preference directly (null / no - english)